lib/r10k/cli/version.rb

```diff
@@ -12,7 +12,16 @@ def self.command
         summary 'Print the version of r10k'
 
         run do |opts, args, cmd|
-          puts R10K::VERSION
+          puts "r10k #{R10K::VERSION}"
+          if opts[:verbose]
+            puts RUBY_DESCRIPTION
+            cmdpath = caller.last.match(/\A(.*#{$PROGRAM_NAME}):/).captures.first
```

Comment:
This could be `caller.last.slice(/\A.*#{$PROGRAM_NAME}/)` since we just want to chop out a substring.